Dockerfile

```diff
@@ -1,30 +1,54 @@
-FROM node:14.16.1-slim
-
-RUN apt-get update && \
-  apt-get upgrade -yq && \
-  apt-get install -yq yarn git zlib1g zlib1g-dev
-
+FROM node:14-alpine as base
```

Comment:
what's the difference between slim and alpine?

Reply:
`alpine` image uses a different linux distribution (Alpine) and size is smaller.

do you think that could be a concern for us?

Reply:
no, it shouldn't be